src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <FeatureGenericMath>false</FeatureGenericMath>
-    <NoWarn>$(NoWarn),1718,SYSLIB0013</NoWarn>
+    <NoWarn>$(NoWarn),1718,SYSLIB0013,IL3051</NoWarn>
```

Comment:
Does this need a `TODO-LLVM`? Seems like a "need to merge more of upstream" issue.

Reply:
I hope that's what it is.  Runs fine upstream, but the warning looks genuine and the test classes are not annotated upstream either, so not sure how it works.

Reply:
Indeed, not totally clear what is up here. Don't think there's much point in sinking time into this, though.